src/main.rs

```diff
@@ -36,6 +39,42 @@ fn read_config() -> serde_json::Value {
         .expect("Failed to parse config file as a JSON.")
 }
 
+fn parse_updates(command: String) -> (CommandKind, Pipeline) {
```

Comment:
perhaps "parse" is not a good prefix for these functions. they don't do any parsing, really